src/app/shared/components/autocomplete/autocomplete.component.scss

```diff
@@ -0,0 +1,3 @@
+input::-webkit-calendar-picker-indicator {
+  display: none;
```

Comment:
is that needed? Can we rely on govuk lib solely please?
If we think the design are not matching the design system we can have a chat about it.
The components from the design system lib should be ready to be used.

Reply:
The gov uk doesn't have autocomplete among its components. This is a custom component in which I used HTML5 datalist so I had to adjust a little bit the style to be compliant.